.github/workflows/main.yml

```
@@ -63,3 +63,30 @@ jobs:
       - name: Run Go tests w/ `-race`
         if: ${{ runner.os == 'Linux' }}
         run: go test -race $(go list ./... | grep -v third_party/)
+  e2e-tests:
```

Let's separate this job definition from the previous one:

```suggestion


  e2e-tests:
```